app/models/openscap_result.rb

```diff
@@ -35,6 +35,7 @@ def create_rule_results
   end
 
   def create_results(rule_results, benchmark_items)
+    openscap_rule_results.delete_all
```

Comment:
@Fryguy @blomquisg @kbrock is there anything nicer that we can use for this pattern of "replace all the results"? (Delete all existing ones, add all the new ones). Should we be concerned about performances? Any suggestion/comment?